git_lint_branch/main.py

```diff
@@ -4,21 +4,23 @@
 from git_lint_branch.linter_output import *
 from git_lint_branch.single import single_linters
 
+
 app = typer.Typer()
 
+
 @app.command()
 def main(upstream: str):
     """
     Lints the commit history reachable from the current HEAD that is not
     on UPSTREAM (i.e., the current branch).
     """
-    repo = Repository('.git')
+    repo = Repository(".git")
```

Comment:
Looks like your editor changes single quotes to double?

Reply:
Yup. It runs `black` on python repos by default. Let me revert this.

Reply:
I happen to prefer single quotes for a specific reason:
```
>>> "this is a string"
'this is a string'
```
they're used in the default `__repr__` for strings. The exception is docstrings where some PEP mandates `"""`, I believe